spec.bs

```diff
@@ -1558,16 +1559,16 @@ To <dfn>generate and score bids</dfn> given an [=auction config=] |auctionConfig
             1. If |originalAds| is not null:
               1. Set |ig|'s [=interest group/ads=] to a new [=list=] of [=interest group ad=].
               1. [=list/For each=] |ad| in |originalAds|:
-                1. If [=query ad k-anonymity count=] given |ig| and |ad|'s
-                  [=interest group ad/render url=] returns true, [=list/append=] |ad| to |ig|'s
-                  [=interest group/ads=].
+                1. Compute |adHashCode| by following [=compute the key hash of ad=] given |ig| and |ad|.

```

Comment:
following? Can we just say "by getting the result of `[=computing the key hash of ad=] [...]`

Reply:
Done